Assignment5.txt

```diff
@@ -0,0 +1,5 @@
+1a) Passing by value makes a copy of the variable which it gives or "passes" to the function. Therefore, the value is the same, but the identity of the variable is different. Passing by value can be compared to an analogy where you want to share a web page with a friend and you give them a printed-out copy, or a disconnected copy of the original. You won't see any of the changes to the web page if they are made, but you can scribble or modify your copy without the changes showing up on the web page. If you decide to destroy your print-out, YOUR copy may be destroyed, but the original web page remains unaffected.
+1b) Passing by reference can also be compared to the analogy where you want to share a web page with a friend, and you send them the URL.The web page that you and your friend are looking at would be identical, even if there are any changes made. If one of you happens to delete the URL, you are deleting your reference to that page, and the page itself would remain perfectly intact. 
+2) The function of the return statement in C is to end the execution of a function and to "return" control to the calling of the function, while also specifying a value to sent to the caller function. This value becomes the value of the function call expression.
```

Comment:
Right! Again, please cite sources that you use. Your wording and explanations are very similar to a lot of the online material readily available.